shibata added the new strings in the Japanese version to the English version. (#83)
fujitake/vantiq-related: vantiq-external-services-integration/docs/en/vantiq-aws-dynamodb.md

![picture20](../../imgs/vantiq-aws-dynamodb/picture20.png)

### Others

#### For configuring TTL

> In the case of DynamoDB, records cannot be deleted in bulk. So it is necessary toconfigure the TTL to have unnecessary records deleted periodically.

Reviewer: ```suggestion
In the case of DynamoDB, records cannot be deleted in bulk.  Set the TTL to have unnecessary records deleted periodically.
```

> The following code example calculates the unix epoch time 30 days after the current date and adds it as a `ttl` column. It is necessary to configure the TTL accordingly on the DynamoDB side.

Reviewer: ```suggestion
The following code example calculates the unix epoch time which points to 30 days after the current timestamp and adds as `ttl`.  It is necessary to configure the TTL accordingly on the DynamoDB side.
```
